### PR TITLE
fix(core,phrases): add notice that protected app is cloud only

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Protected App is not available in open source version, add this notice in openapi docs and error messages.

## Testing

N/A

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

File: packages/phrases/src/locales/en/errors/application.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ const application = {
   user_consent_scopes_not_found: 'Invalid user consent scopes.',
   consent_management_api_scopes_not_allowed: 'Management API scopes are not allowed.',
   protected_app_metadata_is_required: 'Protected app metadata is required.',
-  protected_app_not_configured: 'Protected app provider is not configured.',
+  protected_app_not_configured:
```
Comment (charIeszhao) [inline packages/phrases/src/locales/en/errors/application.ts:11]: Where is this key used?

Comment (wangsijie) [reply packages/phrases/src/locales/en/errors/application.ts:11]: You can search the code, when the protected app is not configured (system table)

**Review by simeng-li — APPROVED** [review]
LGTM.

File: packages/core/src/routes/applications/application-protected-app-metadata.openapi.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "/api/applications/{id}/protected-app-metadata/custom-domains": {
       "get": {
         "summary": "Get the list of custom domains of the protected application.",
-        "description": "Get the list of custom domains of the protected application.",
+        "description": "Get the list of custom domains of the protected application. This feature is not available for open source version.",
```
Comment (simeng-li) [inline packages/core/src/routes/applications/application-protected-app-metadata.openapi.json:6]: nit: maybe we could have some cloud-only tag for these API

**Review by gao-sun — APPROVED** [review]

**Pull request merged**